src/rqt_console/console.py

```diff
@@ -82,6 +83,8 @@ def __init__(self, context):
 
         self._subscriber = None
         self._topic = '/rosout_agg'
+        self._node = rclpy.create_node('rqt_console')
```

Comment:
This should make use of the new ros2_plugin_context._node

Reply:
done